translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
     </message>
     <message>
         <source>Envelope Postfix</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Postfix</translation>
     </message>
     <message>
         <source>100 x 150mm</source>
-        <translation type="unfinished"></translation>
+        <translation>100 x 150 mm</translation>
     </message>
     <message>
         <source>89 x 89mm</source>
-        <translation type="unfinished"></translation>
+        <translation>89 x 89 mm</translation>
     </message>
     <message>
         <source>100 x 200mm</source>
-        <translation type="unfinished"></translation>
+        <translation>100 x 200 mm</translation>
     </message>
     <message>
         <source>Envelope Chinese #10</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 10</translation>
     </message>
     <message>
         <source>Chinese 16k</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesisch 16k</translation>
     </message>
     <message>
         <source>Envelope Chinese #1</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 1</translation>
     </message>
     <message>
         <source>Envelope Chinese #2</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 2</translation>
     </message>
     <message>
         <source>Chinese 32k</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesisch 32k</translation>
     </message>
     <message>
         <source>Envelope Chinese #3</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 3</translation>
     </message>
     <message>
         <source>Envelope Chinese #4</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 4</translation>
     </message>
     <message>
         <source>Envelope Chinese #5</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 5</translation>
     </message>
     <message>
         <source>Envelope Chinese #6</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 6</translation>
     </message>
     <message>
         <source>Envelope Chinese #7</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 7</translation>
     </message>
     <message>
         <source>Envelope Chinese #8</source>
-        <translation type="unfinished"></translation>
+        <translation>Chinesischer Umschlag Nr. 8</translation>
     </message>
     <message>
         <source>ROC 16k</source>
-        <translation type="unfinished"></translation>
+        <translation>ROC 16k</translation>
```

Comment:
Looks unfinished

Reply:
yes, but it is ready. It is also a paper format. See here: https://questionanswer.io/what-is-roc-16k-paper-size/#What_does_16K_paper_mean
Maybe it is more understandable if you change it to paper format ROC 16k?

Reply:
I was thinking that since you were translating e.g. Chnese, why not translate ROC(Taiwan) too